packages/doenetml-worker-rust/lib-doenetml-core/src/core/state_var_interfaces/boolean_state_var_interfaces.rs

```diff
@@ -144,25 +164,26 @@ impl StateVarInterface<bool> for GeneralBooleanStateVarInterface {
         }
     }
 
-    fn calculate_state_var_from_dependencies(&self, state_var: &StateVarMutableView<bool>) {
+    fn calculate_state_var_from_dependencies(&self) -> StateVarCalcResult<bool> {
         if self.have_invalid_combination {
-            state_var.set_value(false);
+            StateVarCalcResult::Calculated(false)
```

Comment:
You're using a mix of implicit returns and explicit ones. Please be consistent, especially in deeply nested structures.

Reply:
I prefer explicit return in this case, but clippy is weird and randomly chooses some to say they should be implicit. Turning off this clippy warning for this function.